client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/services/SubmissionService.kt

```diff
@@ -31,7 +37,24 @@ internal class SubmissionService {
     private fun migrateRequest(request: MigrationRequest) {
         val sourceClient = bioWebClient(request.source, request.sourceUser, request.sourcePassword)
         val targetClient = bioWebClient(request.target, request.targetUser, request.targetPassword)
-        targetClient.submitExt(migratedSubmissions(sourceClient.getExtByAccNo(request.accNo), request.targetOwner))
+        val extSerializer = extSerializationService()
+        val source = sourceClient.getExtByAccNo(request.accNo)
+        val migrated = migratedSubmissions(source, request.targetOwner)
+        val fileLists = source.allFileList.map { asTempFile(it, sourceClient, request.tempFolder, extSerializer) }
```

Comment:
maybe
```
        val files = source.allFileList
            .map { File(request.tempFolder, it.fileName) to sourceClient.getReferencedFiles(it.filesUrl!!) }
            .onEach { (file, files) -> file.writeText(extSerializer.serialize(files)) }
            .map { it.first }       
````

as passing services as parameter looks exrange for me, also we may consider transform extSerializationService into a class field.